src/lapack/util/app/qut/front/flamec/test/flash_sm/test_Apply_Q.c

```diff
@@ -188,7 +188,8 @@ int main(int argc, char *argv[])
       FLASH_Copy( A_save, A );
 
       FLASH_Obj_flatten( A, A_flat );
-      FLA_QR_UT( A_flat, t, T_flat );
```

Comment:
intentional change?

Reply:
At least, leaving the old line in was. It was complaining about too many args. Since I don't have this test working correctly, I can't verify I removed the correct arg yet. But this gets the test compiling.